### PR TITLE
Add test for host with .ics domain name

### PR DESCRIPTION
**Pull request by sambible**
### Problem Statement
This is testing a case where hosts assigned a domain ending in .ics cause Satellite to error out on old and new UI when navigating to the details page. 

### Related Issues
https://issues.redhat.com/browse/SAT-26202

### PRT 
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k 'test_read_host_with_ics_domain'

### CODE REVIEW

**Comment by sambible** [comment]
trigger: test-robottelo
pytest: tests/foreman/ui/test_host.py -k 'test_read_host_with_ics_domain'

File: tests/foreman/ui/test_host.py [hunk 1]
```diff
@@ -257,6 +257,34 @@ def test_positive_read_from_details_page(session, module_host_template):
         assert 'Admin User' in values['properties']['properties_table']['Owner']
 
 
+def test_read_host_with_ics_domain(
+    session, module_host_template, module_location, module_org, target_sat
+):
+    """Create new Host with ics domain name and verify that it can be read
+
```
Comment (ColeHiggins2) [inline tests/foreman/ui/test_host.py:264]: Can we clean up the doc-string and add :steps: and :customerscenario: if its a customer scenario ?

**Review by damoore044 — APPROVED** [review]
Nice, looks good pending @ColeHiggins2 's comment

File: tests/foreman/ui/test_host.py [hunk 2]
```diff
@@ -257,6 +257,34 @@ def test_positive_read_from_details_page(session, module_host_template):
...
+    :id: 54e3db92-16c2-412b-bf68-44d479c5987b
+
+    :expectedresults: Host ending with ics domain name can be accessed through Host UI
+    """
```
Comment (vijaysawant) [inline tests/foreman/ui/test_host.py:268]:
```suggestion
    :Verifies: SAT-26202
    """
```

File: tests/foreman/ui/test_host.py [hunk 3]
```diff
@@ -257,6 +257,34 @@ def test_positive_read_from_details_page(session, module_host_template):
         assert 'Admin User' in values['properties']['properties_table']['Owner']
 
 
+def test_read_host_with_ics_domain(
+    session, module_host_template, module_location, module_org, target_sat
```
Comment (vijaysawant) [inline tests/foreman/ui/test_host.py:261]:
```suggestion
    session, module_host_template, module_location, module_org, module_target_sat
```

File: tests/foreman/ui/test_host.py [hunk 4]
```diff
@@ -257,6 +257,34 @@ def test_positive_read_from_details_page(session, module_host_template):
...
+    template.name = gen_string('alpha').lower()
+    ics_domain = target_sat.api.Domain(
+        location=[module_location],
+        organization=[module_org],
+        name=gen_string('alpha').lower() + '.ics',
+    ).create()
+    template.domain = ics_domain
+    host = template.create()
+    host_name = host.name
+    with session:
```
Comment (vijaysawant) [inline tests/foreman/ui/test_host.py:279]:
```suggestion
    with module_target_sat.ui_session() as session:
```

File: tests/foreman/ui/test_host.py [hunk 5]
```diff
@@ -257,6 +257,34 @@ def test_positive_read_from_details_page(session, module_host_template):
...
+):
+    """Create new Host with ics domain name and verify that it can be read
+
+    :id: 54e3db92-16c2-412b-bf68-44d479c5987b
+
+    :expectedresults: Host ending with ics domain name can be accessed through Host UI
+    """
+    template = module_host_template
+    template.name = gen_string('alpha').lower()
+    ics_domain = target_sat.api.Domain(
```
Comment (vijaysawant) [inline tests/foreman/ui/test_host.py:271]:
```suggestion
    ics_domain = module_target_sat.api.Domain(
```

**Review by vijaysawant — CHANGES_REQUESTED** [review]

File: tests/foreman/ui/test_host.py (hunk 2 above)
Comment (sambible) [reply tests/foreman/ui/test_host.py:268]: Thanks, I keep forgetting how to do this lol

**Comment by ColeHiggins2** [comment]
Merge pending @vijaysawant Comments

**Pull request merged**